auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/external/DeltaLoginController.kt

```diff
@@ -50,6 +51,11 @@ class DeltaLoginController(
             logger.info("Invalid parameters for login request, redirecting back to Delta")
             return call.respondRedirect(deltaConfig.deltaWebsiteUrl + "/login?error=delta_invalid_params&trace=${call.callId!!.encodeURLParameter()}")
         }
+        if (params.timestamp?.let { it < (System.currentTimeMillis() / 1000) - 1.hours.inWholeSeconds } == true) {
```

Comment:
Could you make a const for the expiry time so that if it gets changed it is changed here and wherever it is checked on post requests?

Reply:
It's Delta that checks it when the user is redirected back after logging in. TBH I don't know how long they're even valid for, it's checked by Spring Security somewhere.

I'll pull out a constant anyway